build/crd-sync.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail  # exit on errors and unset vars, and stop on the first error in a "pipeline"
+
+BRANCH="${BRANCH:-main}"
+
+mkdir -p .go
+
+# Clone repositories containing the CRD definitions
+for REPO in cert-policy-controller config-policy-controller iam-policy-controller governance-policy-propagator
+do
+    git clone -b "${BRANCH}" --depth 1 https://github.com/stolostron/${REPO}.git .go/${REPO}
+done
+
+(
+    pwd
```

Comment:
Is this intentional?

Reply:
No, thanks for catching it. It's a leftover from when I had the wrong subshell syntax